src/redux/features/plantings/plantingsSlice.ts

```diff
@@ -8,7 +8,7 @@ export type PlantingSearchData = {
   delivery: { withdrawal_id: string };
   'numPlants(raw)': string;
   plantingSite: { id: string };
-  plantingSubzone?: { id: string; 'totalPlants(raw)': number };
+  plantingSubzone?: { id: string; totalPlants: string };
```

Comment:
using string value so it comes pre-formatted/localized.. we won't be using raw numbers for math.
Also, these types should maybe be moved into src/types?